src/action_install.rs

```diff
@@ -54,6 +56,10 @@ pub fn install(targets: &[String], rua_paths: &RuaPaths, is_offline: bool, asdep
 		is_offline,
 		asdeps,
 	);
+
+	let folder_deleter = Box::new(FolderDeleter::new());
+	let left_overs_deleter = LeftOversDeleter::new(folder_deleter);
+	left_overs_deleter.delete_folders(targets, rua_paths);
```

Comment:
Hey, thanks for the PR!

One question on the libraries, what do you think of using `rm_rf` library for directory removal? It's a library that I created specifically to remove files and directories where `fs::remove_dir_all` would fail (it sometimes happens with AUR build dirs). This library is already used in a few places here, it's well tested as well. Then the only thing left would be to just call this function in a few places after the build process is completed and built artifacts are copied to `checked_tars` dir?

Reply:
Thank you for being gentle! :-)
I will think over what you are suggesting, it looks logical. I will need a few days of meditation on that.

Reply:
> Then the only thing left would be to just call this function in a few **places**

Why plural **places** and not singular place? As far as I understand from the code, its only a single place, right at the end of function `install` in file `action_install.rs`, correct?

Reply:
@refaelsh Ah yes, indeed, you're right. Here it'll only be in one place, after the packages are moved to the `checked_tars` directory. It will also be needed for the potential `clean-cache` command, but that's a separate thing and can be added independently